Code/Framework/AzCore/AzCore/IO/Streamer/StreamerContext.cpp

```diff
@@ -269,16 +284,49 @@ namespace AZ
 
         void StreamerContext::CollectStatistics(AZStd::vector<Statistic>& statistics)
         {
-            statistics.push_back(
-                Statistic::CreateInteger(ContextName, "Pending requests", aznumeric_caster(m_preparedRequests.size())));
+            statistics.push_back(Statistic::CreateInteger(
+                ContextName, "Pending requests", aznumeric_caster(m_preparedRequests.size()),
+                "The number of requests that are waiting to be queued for processing. These are the requests that can still be scheduled. "
+                "Larger numbers allow the scheduler to do more optimizations."));
 #if AZ_STREAMER_ADD_EXTRA_PROFILING_INFO
-            statistics.push_back(Statistic::CreateFloat(ContextName, PredictionAccuracyName, m_predictionAccuracyUsStat.GetAverage() / 1000.0));
-            statistics.push_back(Statistic::CreatePercentage(ContextName, LatePredictionName, m_latePredictionsPercentageStat.GetAverage()));
-            statistics.push_back(Statistic::CreatePercentage(ContextName, MissedDeadlinesName, m_missedDeadlinePercentageStat.GetAverage()));
+            statistics.push_back(Statistic::CreateTimeRange(
+                ContextName, PredictionAccuracyName, m_predictionAccuracyStat.CalculateAverage(), m_predictionAccuracyStat.GetMinimum(),
+                m_predictionAccuracyStat.GetMaximum(),
+                "The delta between time the scheduler predicated the request would be done and the actual time the request completed in "
+                "milliseconds. The smaller this value, the more accurate the scheduler is."));
```

Comment:
Is this an absolute value or can it be negative?
If the values can be negative, I would say that "the closer the value is to zero, the more accurate the scheduler is".

Reply:
The time is always positive. The rate of early or late predictions (which would be the difference between a positive or negative value) is tracked in a separate stat. This was done to avoid the prediction to be working well in case there's a combination of many large early and late predictions.